### ISSUE TITLE
Looking for a maintainer

### ISSUE THREAD

**Issue by themisir** [hivedb/hive#966]
tldr: We are looking for a new maintainer to provide support to hive project. If you're interested please comment bellow.

A bit of backstory if you're interested

I (@themisir) was doing some light maintenance for last year and a few months maybe. I wanted to keep hive alive and provide stability to existing feature set, because back in time I have been running a small company that was using hive on its mobile apps, and I thought why not provide support instead of moving away, actually eventually I moved away from hive on those apps to store data on server side, but I still wanted to provide support to give back some value to the community I'm myself getting value from. But we had to close that company alongside with apps and a few months ago I moved away to work on another company with different tech stacks. After moving away I lost my interested in flutter ecosystem as I am mainly working on much low level programming with different toolset, so did my interest in working on this project.

Hive was actually pretty interesting project for me, It had taught me really interesting concepts, and forced me to explore more about stuff like serialization and working with binary data, data recovery, encryption and other "fun" stuff. But that's not enough to keep myself engaged with this project and I no longer able to provide required support to this project.

I am sorry for not being able to provide value this project deserves, so after talking to Simon we decided to look for another maintainer. If you're interested, you think you can provide some value please comment bellow.

My personal suggestions to the future maintainer:

- Please make sure API stays stable, If you need hive-2.0 call it something else and release as a separate project, but I don't think any user would enjoy process of migrating from one database to another one. Even if you keep core features same, changing a simple API will break someone else's workflow, so be sure to keep public api stable. [check this out](https://youtu.be/Pzl1B7nB9Kc?t=174) also [related xkcd](https://xkcd.com/1172/)
- You don't have to add new features to keep hive alive. I know a new shinny feature will boost popularity, but I think as a database hive is currently in a good spot right now. It can store data, and it can serialize complex objects. If you want to implement a new feature (eg let's say cloud sync), adding it as an extension / additional package would be more fit than modifying core package. So users can able to opt-in / opt-out if they want to. 

Please note that those are just my personal thoughts, not any sort of strong rules to follow.

**Comment by TheOneWithTheBraid** [comment]
How can you apply as maintainer? I could imagine to do this together with other people. For me, Hive is the foundation of so many Flutter projects; one of those things making Flutter and Dart the unique they are. I would love to keep maintaining this in order to ensure Hive keeps providing its exceptional conveyance.

**Comment by themisir** [comment]
> How can you apply as maintainer? I could imagine to do this together with other people. For me, Hive is the foundation of so many Flutter projects; one of those things making Flutter and Dart the unique they are. I would love to keep maintaining this in order to ensure Hive keeps providing its exceptional conveyance.

That would be awesome, let's start by merging your changes and I'll ping you if something breaks related to that patch. I will probably still provide some sort of support & be there to discuss changes, or can fix some issues.

cc: @leisim can you give @TheOneWithTheBraid contributor access please?

**Comment by leisim** [comment]
Invite sent 🙌

**Comment by ahmednfwela** [comment]
@themisir I would like to apply for maintainer as well, I use Hive in all of my projects, and I have some improvements in mind (especially for the generator)

**Comment by AlexMainaMunyua** [comment]
I have read your story and it is very interesting. I haven't used Hive before, But I want to make a commitment to help you maintain it. I will go through the issues and in case I spot any contribution I can make, I will raise a PR or respond to the issues directly.

**Comment by AmrSaied** [comment]
this is the third party I do it above hive 
https://pub.dev/packages/hive_ui/
A Flutter package that allows you to easily work with Hive databases. With this package, you can explore all database boxes, edit table rows, add new rows to tables, search boxes by column name and value, delete rows or all data from a box, copy selected values and select a specific color for the Hive UI view.

**Comment by loonix** [comment]
just wonder why @AmrSaied you took out the repository when I liked it :)

**Comment by vtisnado** [comment]
Hello, did you have any luck finding maintainers?
The latest stable version **v2.2.3** is from June 2022 and the version **v3.0.0-dev** is still in **pre-release**.
It would be interesting to have some news about the future plans of the project.
